examples/Scenario4Bundle-jws-payload-expanded.json

```diff
@@ -0,0 +1,75 @@
+{
+  "iss": "https://spec.smarthealth.cards/examples/issuer",
+  "nbf": 1628870759,
+  "vc": {
+    "type": [
+      "https://smarthealth.cards#health-card",
+      "https://smarthealth.cards#immunization",
+      "https://smarthealth.cards#mmr",
```

Comment:
I'm wondering if we want to define a type for the combination or list them all separately in order to more easily accommodate a) cards that have individual vaccinations, not the combo and b) being able to more specifically request e.g. rubella vaccination info.

Reply:
I now see your way of creating these in https://github.com/smart-on-fhir/health-cards/pull/169/files . It's a pragmatic solution and possibly granular enough.